tests/syscall/getrandom/host/host.c

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/host.h>
+#include <openenclave/internal/error.h>
+#include <openenclave/internal/tests.h>
+#include <stdio.h>
+#include "getrandom_u.h"
+
+int main(int argc, const char* argv[])
+{
+    oe_result_t result = OE_UNEXPECTED;
+    oe_enclave_t* enclave = NULL;
+
+    if (argc != 2)
+    {
+        fprintf(stderr, "Usage: %s ENCLAVE_PATH\n", argv[0]);
+        return 1;
+    }
+
+    const uint32_t flags = oe_get_create_flags();
+
+    result = oe_create_getrandom_enclave(
+        argv[1], OE_ENCLAVE_TYPE_SGX, flags, NULL, 0, &enclave);
```

Comment:
```suggestion
        argv[1], OE_ENCLAVE_TYPE_AUTO, flags, NULL, 0, &enclave);
```

I don't see anything SGX-specific in this test